Gemfile.lock

```diff
@@ -139,4 +143,4 @@ RUBY VERSION
    ruby 2.6.2p47
 
 BUNDLED WITH
-   2.0.1
+   2.1.4
```

Comment:
We cannot use this bundler, it must be bundled with 2.0.1

Reply:
Yep, I had this pinned in another branch, but I had to base off `current` and rebase, fixed now.